woocommerce/woocommerce-ios
# [Deposit Summary] Show collapsed Deposit Summary until tapped

Closes: #11224


## Description


The Deposits Summary is fairly large, and most of the time would require a user to scroll to do whatever they came to the Payments menu for.

The key information is the available and pending fund totals.

This PR collapses the view when the Payments Menu is loaded, and allows it to be expanded with a tap.

## Testing instructions


1. Launch the app
2. Switch to a WooPayments store with only one currency set up
3. Navigate to `Menu > Payments`
4. Observe that after a few seconds, the Deposit Summary is shown, collapsed.
5. Tap the expansion button
6. Observe that the Deposit Summary is shown in full

Repeat the above test with a multi-currency WooPayments account. (I can provide a login if needed)
Observe that the currency tabs are hidden when it's collapsed.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/2472348/c633581b-9705-4ca8-b116-30486eb7e2b5



---
